api/src/wfl/executor.clj

```diff
@@ -138,8 +138,7 @@
 (defn ^:private create-user-comment
   "Create a user comment to be added to an executor submission."
   [note {:keys [uuid] :as _workload} snapshot-id]
-  (str/join \space [note "Workload:" uuid "Snapshot ID:"
-                    snapshot-id "origin:" (env/getenv "WFL_WFL_URL")]))
+  (str/join \space [note "Workload:" uuid "Snapshot ID:" snapshot-id]))
```

Comment:
I removed the `WFL_WFL_URL` origin because this isn't populated correctly in `gotc-prod-wfl` (defaults and points to `gotc-dev-wfl`).

Reply:
Thanks. And it is consistent for a deployment.